src/test/java/tc/AddictionTest.java

```diff
@@ -0,0 +1,31 @@
+package tc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tc.Operations;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import tc.DataProviders;
+
+public class AddictionTest {
+    private Operations operations;
+
+    @BeforeMethod(groups = {"AddSubOperations"})
+    public void before() {
+        operations = new Operations();
+    }

```

Comment:
Looks like you invoke wrong calculator here. Why do you do not use calculator from jar file?

Reply:
I used calculator methods in Operations class to create my own methods for add,sub,div and Multiply